index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Tribute Page</title>
+</head>
+
+<body>
+    <div id="main">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+        <div id="img-div"><img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" id="image">
+            <div id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</div>
+        </div>
+        <div>
+            <div>
+                <h3 id="tribute-info">Here's a time line of Dr. Borlaug's life:</h3>
+            </div>
+            <div>
+                <ul>
+                    <li><b>1932</b> - Born in Cresco, Iowa</li>
+                    <li><b>1933</b> - Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program known as the "National Youth Administration"</li>
+                    <li><b>1935</b> - Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping starving Americans. "I saw how f ood changed them", he said. "All of this left scars on me."</li>
+                    <li><b>1937</b> - Finishes university and takes a job in the US Forestry Service</li>
+                    <li><b>1938</b> - Marries wife of 69 years Margret Gibson. Gets laid off due to budget cuts. Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding pest-resistent plants.</li>
+                </ul>
+            </div>
+        </div>
```

Comment:
Try to use a section tag because the section tag represents a generic standalone section of a document, which doesn't have a more specific semantic element to represent it. Sections should always have a heading, with very few exceptions.

Reply:
Nice job explaining why does he need the changes but please ask for any change in a more friendly and polite way by saying "Please add ..." or "Kindly add ...".

Reply:
Thank you very much